casper/src/test/scala/coop/rchain/casper/util/rholang/RegistrySigGen.scala

```diff
@@ -0,0 +1,107 @@
+package coop.rchain.casper.util.rholang
+
+import com.google.protobuf.ByteString
+
+import coop.rchain.casper.protocol.DeployData
+import coop.rchain.crypto.codec.Base16
+import coop.rchain.crypto.hash.Blake2b512Random
+import coop.rchain.crypto.signatures.{Ed25519, Secp256k1}
+import coop.rchain.models.Expr.ExprInstance.GInt
+import coop.rchain.models.Expr.ExprInstance.{GBool, GByteArray, GString}
+import coop.rchain.models.rholang.implicits._
+import coop.rchain.models.{Bundle, ETuple, Expr, GPrivate, Par}
+import coop.rchain.rholang.interpreter.{PrettyPrinter, Runtime}
+
+case class InsertSigned(pk: Hex, value: (Long, Contract), sig: Hex) {
```

Comment:
wart remover will complain about non-final case classes

Reply:
is it complaining for a good reason? Or should I add an annotation to suppress the warning?

Reply:
oh... it doesn't mind that I'm overriding toString... just that I'm missing `final`. Fair enough.